### PR TITLE
Adding template to execute test container for performance Testing

### PR DESCRIPTION
**Pull request by sumitnagal**
This is well-proven solution we are using in Intuit, and we have not contributed back to open source, hence adding as part GED-2020 project.

### CODE REVIEW

File: templates/perf-infra/manifests.yaml [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# Stress test to test upper bounds of concurrent pods
```
Comment (alexec) [inline templates/perf-infra/manifests.yaml]: My only change is that everything should go into the manifest. Including any docs.

**Review by alexec — CHANGES_REQUESTED** [review]
Can you please run make to generate the HTML?

**Comment by sumitnagal** [comment]
I am not able to make in hello template itself , let me know if I am missing anything 
image

**Comment by alexec** [comment]
You need to run `make` in the root directory of the project.

**Comment by sumitnagal** [comment]
Added changes 
➜  argo-workflows-catalog git:(master) ✗ make
go run ./hack
templates/argocd-sync-and-wait/manifests.yaml
templates/buildkit/manifests.yaml
templates/distro/manifests.yaml
templates/go-build/manifests.yaml
templates/hello-world/manifests.yaml
templates/jenkins-pipeline/manifests.yaml
templates/sendmail/manifests.yaml
templates/slack/manifests.yaml
templates/terraform-basic/manifests.yaml

**Comment by sumitnagal** [comment]
@alexec , please let me know

**Review by alexec — APPROVED** [review]

**Pull request merged**